### DISCUSSION TITLE
Add Profiles (similar to armory crate)

### DISCUSSION THREAD

**Discussion by 2ixks** [seerge/g-helper#2950] [Ideas]
Instead of switching between _fan presets constantly on each application_, it would be better having a **profiles for seperate/multiple applications so that when the specific app opens (that you selected for profiles), it would auto-set the fan control, rgb, etc.**. It's often used by many people when they don't want to switch presets constantly.

**Comment by eried** [comment]
I think the problem is how to detect which application is the active, dont you experience issues with the control in armoury crate because of this? like if the game has a launcher, etc
